### ISSUE TITLE
i need help with ublocking apps it keeps saying Initializing.

### ISSUE THREAD

**Issue by gcs2209** [uBlockOrigin/uAssets#19212]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

i need help with ublocking apps it keeps saying Initializing. i tried to unblock tiktok, snapchat and instagram but it wont let me 

### URL(s) where the issue occurs.

https://tikotk.com
https://snapchat.web
https://instagram.com

### Screenshot(s)

![Screenshot 2023-07-29 4 34 50 PM](https://github.com/uBlockOrigin/uAssets/assets/140877171/b3e2311f-cfbf-4e36-bb80-0b9329cbcfe4)

### Configuration

<details>

```yaml
uBlock Origin 1.50.0:
 since: 42d.10h.16m
Chromium: 115
filterset (summary):
 network: 97543
 cosmetic: 39512
 scriptlet: 18109
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  easylist: 68894-9, 2d.22h.34m
  easyprivacy: 33341-62, 5d.7h.32m
  plowe-0: 3720-1071, 4d.29m
  ublock-badware: 7572-142, 4d.28m
  ublock-filters: 34110-219, 4d.27m
  ublock-privacy: 441-5, 1d.20h.15m
  ublock-quick-fixes: 283-43, 10m
  ublock-unbreak: 2088-32, 4d.1m
  urlhaus-1: 6447-0, 10m
filterset (user): [empty]
trustedset:
 added: [array of 7 redacted]
 removed:
  chrome-extension-scheme
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 6193 ms (selfie)
 maxAssetCacheWait: 223 ms
```
</details>

**Comment by okiehsch** [comment]
I don't understand the issue.
You mean you can't use the apps on your mobile device and think it is caused by uBO?
Or do you mean that the websites don't work on your end?

**Comment by iam-py-test** [comment]
> `tikotk.com`

Do you mean tiktok[.]com, as `tikotk.com` is parked
